larray/core/session.py

```diff
@@ -1392,6 +1393,155 @@ def display(k, v, is_metadata=False):
         return res
 
 
+# XXX: I wonder if we shouldn't create an AbstractSession instead of defining the _disabled()
+#      private method below.
+#      Auto-completion on any instance of a class that inherits from FrozenSession
+#      should not propose the add(), update(), filter(), transpose() and compact() methods
```

Comment:
**IF** we keep that "feature" of having those functions be non functional, then I agree with you, they should not be available at all **BUT** I am unconvinced about limiting so much what users can do (see below for more details)